qa/rpc-tests/maxuploadtarget.py

```diff
@@ -175,13 +175,13 @@ def run_test(self):
         getdata_request = msg_getdata()
         getdata_request.inv.append(CInv(2, big_old_block))
 
-        max_bytes_per_day = 200*1024*1024
-        daily_buffer = 144 * MAX_BLOCK_SIZE
+        max_bytes_per_day = 800*1024*1024
+        daily_buffer = 144 * 4000000
```

Comment:
@sipa 👍 on a new constant to make things readable, but sorry I'm just starting to learn the source code here so I still have simple questions.  Is the reason your setting this different than the MAX_BLOCK_SIZE because segwit is supposed to help double the amount of data per block?  I would think this would be 4MB not 4_000_000 which is close... Makes me think the test would be MAX_BLOCK_SIZE * 2.  My main misunderstanding is why this value is changing from max constant when the test appears to be testing mining full blocks... Hopefully that helps clear up my confusion.